### PR TITLE
DEV-3818-exam-03-drop-the-thread-remake-exercise

### PR DESCRIPTION
**Pull request by mikysett**

### CODE REVIEW

File: subjects/drop_the_blog/README.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  - `drops`: that will save the number of dropped articles.
+  - `states`: that will save the state of multiple articles. If the article is not dropped, the state will be `false`, otherwise it will be `true`.
+- `Article`: containing:
+  - `id` as `usize`.
+  - `body`: as `String`.
+  - `parent`: a link to the structure `Blog`. (Tip: this should be a reference).
+
+You'll need to also add the following associated functions to the structures:
+
+- `Blog`:
```
Comment (nprimo) [inline subjects/drop_the_blog/README.md:17]: Missing a new line here to be `prettier` complaint

**Review by nprimo — CHANGES_REQUESTED** [review]
There is a small fix for `prettier` - besides that, the subject looks fine and in line with the other exercise.

File: subjects/drop_the_blog/README.md (hunk 1 above)
Comment (mikysett) [reply subjects/drop_the_blog/README.md:20]: done. Also added extra empty lines and removed `:` at the start of elements' list for more consistency.

**Review by nprimo — APPROVED** [review]
LGTM!

**Pull request merged**